modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -43,6 +43,13 @@
           :selected-items="items"
         />
       </div>
+
+      <AposRelationshipFieldsManager
+        v-if="relationshipSchema"
+        :schema="relationshipSchema"
+        :item="clickedItem"
```

Comment:
Do not pass the whole item, use `v-model` to pass in `clickedItem._fields`.

Reply:
Done